src/util/parented_ptr.h

```diff
@@ -100,10 +100,10 @@ class parented_ptr final {
 };
 
 template<typename... Args>
-concept AnyIsQObject = (... || std::is_base_of_v<QObject, Args>);
+concept AnyIsQObject = (... || std::same_as<QObject*, Args>);
```

Comment:
that won't work almost always because we never pass a literal QObject...
Please try my previous suggestion instead.